src/commands/force/org/open.ts

```diff
@@ -51,7 +50,17 @@ export class OrgOpenCommand extends SfdxCommand {
       return output;
     }
     // we actually need to open the org
-    await this.checkLightningDomain(url);
+    try {
+      this.ux.startSpinner(messages.getMessage('domainWaiting'));
+      await checkLightningDomain(url);
+    } catch (err) {
+      const domain = `https://${/https?:\/\/([^.]*)/.exec(url)[1]}.lightning.force.com`;
+      const timeout = new Duration(new Env().getNumber('SFDX_DOMAIN_RETRY', 240), Duration.Unit.SECONDS);
+      this.logger.debug(`Did not find IP for ${domain} after ${timeout.seconds} seconds`);
```

Comment:
What do you think is the best way to throw the error?
Irrespective of what error we are catching, I am throwing  "domainTimeoutError" as it was used previously. What do you think of this?

Reply:
I think the error is quite descriptive, the function will throw after the timeout+retries end so is safe to say it fits the description, the key `domainTimeoutError` seems descriptive enough to me.